obsidianhtml/__init__.py

```diff
@@ -226,8 +226,10 @@ def ConvertMarkdownPageToHtmlPage(page_path_str, pb, backlinkNode=None):
         if l[0] == '/':
             # Internal link, skip
             continue
-
-        new_str = f"<a href=\"{l}\" target=\"_blank\" class=\"external-link\""
+        external_blank_html = ''
+        if config['toggles']['external_blank']:
```

Comment:
this line will give a key error if someone has an outdated config.yml

default solution is to add a default value (same as in the example config) here:
https://github.com/jcolson/obsidian-html/blob/b8f278f5dc5e4b44dc9bda776384c34c9e8f1847/obsidianhtml/__init__.py#L382-L413